DOCS/man/options.rst

```diff
@@ -7488,20 +7509,27 @@ Miscellaneous
     See ``--audio-display`` how to control display of cover art (this can be
     used to disable cover art that is part of the file).
 
-``--cover-art-auto-exts=ext1,ext2,...``
-    Cover art extentions to try and match when using ``cover-art-auto``.
+``--image-exts=ext1,ext2,...``
+    Image file extentions to try to match when using ``--cover-art-auto``,
+    ``--autocreate-playlist`` or ``--directory-filter-types``.
 
     This is a string list option. See `List Options`_ for details.
 
 ``--cover-art-whitelist=filename1,filename2,...``
     Filenames to load as cover art, sorted by descending priority. They are
-    combined with the extensions in ``--cover-art-auto-exts``. This has no
+    combined with the extensions in ``--image-exts``. This has no
     effect if ``cover-art-auto`` is ``no``.
 
     Default: ``AlbumArt,Album,cover,front,AlbumArtSmall,Folder,.folder,thumb``
 
     This is a string list option. See `List Options`_ for details.
 
+``--video-exts=ext1,ext2,...``
+    Video file extentions to try to match when using ``--autocreate-playlist`` or
+    ``--directory-filter-types``.
+
+    This is a string list option. See `List Options`_ for details.
+
```

Comment:
In original autoload script. Many PR once try to change the value of whitelist.
So could add a tip to get the default value in terminal here.

Reply:
added a note about `--help=video-exts`